HTML Component.flood/main.flp

```diff
@@ -45,6 +45,9 @@ for timeline in timelines
     var cssFolderAbsPath = "\(baseUrl)/\(cssFolderName)"
     var commonFolderAbsPath = "\(baseUrl)/\(commonFolderName)"
 
+    //Relative path to the common folder 
```

Comment:
this is just a style thing, but always add space between the "//" and the comment. So:
`// Relative ...`